node/actors/network/src/proto/gossip.proto

```diff
@@ -19,10 +19,18 @@ message PushValidatorAddrs {
 }
 
 message PushBatchVotes {
+  // Requesting the peer to respond with votes for the batch. 
+  optional roles.attester.Batch want_votes_for = 2; // optional
   // Signed roles.validator.Msg.votes
   repeated roles.attester.Signed votes = 1;
 }
 
+message PushBatchVotesResp {
+  // Signed roles.validator.Msg.votes
+  // Empty if want_snapshot in request was not set.
```

Comment:
```suggestion
  // Empty if want_votes_for in request was not set.
```
Cool, `want_votes_for` sounds like it can be used as an FYI as well so we know what the remote party thinks even without any votes to send where this could be deducted from.

Reply:
done